ZelBack/src/services/fluxService.js

```diff
@@ -1402,6 +1402,33 @@ async function installFluxWatchTower() {
   }
 }
 
+/**
+ * Restart FluxOS via nodemon (executes the command `touch ` on package.json).
+ * @param {object} req Request.
+ * @param {object} res Response.
+ */
+async function restartFluxOS(req, res) {
+  try {
+    const authorized = await verificationHelper.verifyPrivilege('admin', req);
+    if (authorized === true) {
+      const nodedpath = path.join(__dirname, '../../../package.json');
+      const exec = `touch ${nodedpath}`;
```

Comment:
Can you provide more information why it is done this way? How does it work?
If we modify package.json in some way it will cause conflicts on git pulling, updating flux

Reply:
it not modyficate file ;) it touch it and tiggering nodemon to reset .. u can normal git pull

Reply:
The touch command changes certain dates for each file argument. By default, touch sets both the date of last file modification and the date of last file access to the current time.

Reply:
changed to 'pm2 restart flux' command.